MCUIViewLayoutExample/UIViewLayoutExampleTests/MCUIViewLayoutPositionTest.m

```diff
@@ -331,4 +335,25 @@ - (void)testFitWidthWithMargins {
     XCTAssertTrue(rectEquals(110.0f, 1000.0f, 85.0f, 20.0f, rect), @"");
 }
 
+- (void)testCeilFloatToDisplayScale
+{
```

Comment:
Hum, code was written when we where doing bracket on next line

I'll run autoformat and commit after the PR